<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>812</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>791</width>
      <height>371</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Input</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>761</width>
        <height>181</height>
       </rect>
      </property>
      <property name="title">
       <string>Open file</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="gridLayoutWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>731</width>
         <height>90</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="files_selectro_Layout">
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="horizontalSpacing">
         <number>20</number>
        </property>
        <property name="verticalSpacing">
         <number>6</number>
        </property>
        <item row="0" column="1">
         <widget class="QLineEdit" name="SP_line"/>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="openSP_button">
          <property name="text">
           <string>Open</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="SP_label">
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="text">
           <string>Spectrum Folder</string>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="gridLayoutWidget_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>130</y>
         <width>731</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="2">
         <widget class="QRadioButton" name="Chrom_radio_4">
          <property name="text">
           <string>*.raw</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QRadioButton" name="Chrom_radio">
          <property name="text">
           <string>*.csv</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QRadioButton" name="Chrom_radio_3">
          <property name="text">
           <string>*.mzML</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="SP_label_3">
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="text">
           <string>File type</string>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Noise Filter</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>761</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string>Filter</string>
      </property>
      <widget class="QCheckBox" name="cBFilter">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>40</y>
         <width>141</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Enable Filtering</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="gBFilter">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>30</y>
         <width>561</width>
         <height>51</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QDoubleSpinBox" name="IntensePerc_spinbox">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>10</y>
          <width>81</width>
          <height>24</height>
         </rect>
        </property>
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.050000000000000</double>
        </property>
        <property name="value">
         <double>0.800000000000000</double>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>10</y>
          <width>171</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Break Count</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="BreakCount_spinbox">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>10</y>
          <width>81</width>
          <height>24</height>
         </rect>
        </property>
        <property name="maximum">
         <number>300</number>
        </property>
        <property name="value">
         <number>200</number>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>171</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Max Intensity Percentile</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Find method</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>761</width>
        <height>161</height>
       </rect>
      </property>
      <property name="title">
       <string>Open file</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="gridLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>731</width>
         <height>90</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="files_selectro_Layout_2">
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="horizontalSpacing">
         <number>18</number>
        </property>
        <property name="verticalSpacing">
         <number>6</number>
        </property>
        <item row="0" column="2">
         <widget class="QPushButton" name="openDB_button_2">
          <property name="text">
           <string>Open</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="SP_label_2">
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="text">
           <string>Spectrum</string>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="DB_label_2">
          <property name="text">
           <string>DataBase</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="DB_line_2"/>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="SP_line_2"/>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="openSP_button_2">
          <property name="text">
           <string>Open</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>761</width>
        <height>161</height>
       </rect>
      </property>
      <property name="title">
       <string>Open file</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="gridLayoutWidget_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>731</width>
         <height>90</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="files_selectro_Layout_3">
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="horizontalSpacing">
         <number>18</number>
        </property>
        <property name="verticalSpacing">
         <number>6</number>
        </property>
        <item row="0" column="2">
         <widget class="QPushButton" name="openDB_button_3">
          <property name="text">
           <string>Open</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="SP_label_4">
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="text">
           <string>Spectrum</string>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="DB_label_3">
          <property name="text">
           <string>DataBase</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="DB_line_3"/>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="SP_line_3"/>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="openSP_button_3">
          <property name="text">
           <string>Open</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Find It</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>761</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string>Finder</string>
      </property>
      <widget class="QPushButton" name="find_button">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>50</y>
         <width>113</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Find</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="dalton_spinbox">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>67</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="decimals">
        <number>4</number>
       </property>
       <property name="minimum">
        <double>0.000500000000000</double>
       </property>
       <property name="singleStep">
        <double>0.000100000000000</double>
       </property>
      </widget>
      <widget class="QSpinBox" name="ppm_spinbox">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>31</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="value">
        <number>20</number>
       </property>
      </widget>
      <widget class="QRadioButton" name="ppm_radio">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>32</y>
         <width>141</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>ppm Difference</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="dalton_radio">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>68</y>
         <width>151</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Dalton Difference</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>761</width>
        <height>180</height>
       </rect>
      </property>
      <property name="title">
       <string>Adducts selector</string>
      </property>
      <widget class="QRadioButton" name="Positive_radio">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>141</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Positive Spectra</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="Negative_radio">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>141</width>
         <height>17</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Negative Spectra</string>
       </property>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>20</y>
         <width>501</width>
         <height>141</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGroupBox" name="gBPositive">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="title">
           <string>Positive Adducts</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <widget class="QWidget" name="horizontalLayoutWidget">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>20</y>
             <width>481</width>
             <height>31</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QCheckBox" name="cBProtone">
              <property name="text">
               <string>Proton (H+)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="cBSodio">
              <property name="text">
               <string>Sodium (Na+)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="cBPotassio">
              <property name="text">
               <string>Potassium (K+)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="cBAmmonio">
              <property name="text">
               <string>Ammoniun (NH4+)</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="gBNegative">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="title">
           <string>Negative Adducts</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <widget class="QWidget" name="horizontalLayoutWidget_2">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>20</y>
             <width>491</width>
             <height>31</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QCheckBox" name="cBProtone_neg">
              <property name="text">
               <string>Proton -(H+)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="cBCloro">
              <property name="text">
               <string>Cloride (Cl-)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="cBAcetato">
              <property name="text">
               <string>Acetate (CH3COO−)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="cBFormiato">
              <property name="text">
               <string>Formate (HCOO-)</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>670</x>
         <y>20</y>
         <width>71</width>
         <height>61</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Max charge</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="charge_number_comboBox">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <item>
           <property name="text">
            <string>1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>3</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>6</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>7</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>8</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget_3">
       <property name="geometry">
        <rect>
         <x>660</x>
         <y>100</y>
         <width>92</width>
         <height>61</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QPushButton" name="create_button">
          <property name="text">
           <string>CREATE</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
